typescript/infra/src/config/helloworld.ts

```diff
@@ -4,11 +4,13 @@ import { DockerConfig } from './agent';
 
 export interface HelloWorldKathyConfig<Chain extends ChainName> {
   docker: DockerConfig;
-  cronSchedule: string;
   runEnv: string;
   namespace: string;
   chainsToSkip: Chain[];
-  prometheusPushGateway: string;
+  /** How long kathy should take to send a message to all chain pairs before looping (milliseconds) */
```

Comment:
ooh like the ms comment here lol. would be nice for the `messageReceiptTimeout` as well